lib/src/mt_ptp.c

```diff
@@ -759,6 +905,16 @@ static int ptp_init(struct mtl_main_impl* impl, struct mt_ptp_impl* ptp,
   if (ptp->use_pi)
     info("%s(%d), use pi controller, kp %e, ki %e\n", __func__, port, ptp->kp, ptp->ki);
 
+  memset(&ptp->phc2sys.servo, 0, sizeof(struct mt_pi_servo));
+  ptp->phc2sys.realtime_hz = sysconf(_SC_CLK_TCK);
+  ptp->phc2sys.realtime_nominal_tick = 0;
+  if (ptp->phc2sys.realtime_hz > 0) {
+    ptp->phc2sys.realtime_nominal_tick =
+        (1000000 + ptp->phc2sys.realtime_hz / 2) / ptp->phc2sys.realtime_hz;
+  }
+  ptp->phc2sys.stat_sync = false;
+  rte_eal_alarm_set(100000, ptp2sys_handler, ptp);
```

Comment:
Can we directly start the phc2sys directly regardless the ptp stat? Maybe more better to start the timer after ptp is synced.

Reply:
> Can we directly start the phc2sys directly regardless the ptp stat? Maybe more better to start the timer after ptp is synced.

Now, the phc2sys adjustment step has been executed with the ptp adjustment step. phc2sys doesn't run alone anymore.